easybuild/easyconfigs/a/Arrow/Arrow-0.16.0_fix-intel.patch

```diff
@@ -0,0 +1,74 @@
+- avoid "Unknown compiler" CMake error
+- fix compilation error by replacing '__force_inline' (no longer known by Intel compiler) with 'inline'
```

Comment:
Github diff viewer renders this list as if they were removals. I guess this is only cosmetic as they're not in a `---/+++` section?

Reply:
ugh, ok, will change